packages/mui-material/src/Chip/Chip.js

```diff
@@ -137,12 +137,9 @@ const ChipRoot = styled('div', {
           marginLeft: 4,
           marginRight: -4,
         }),
-        ...(ownerState.color !== 'default' && {
-          color: 'inherit',
-        }),
-        ...(ownerState.iconColor &&
+        ...(ownerState.color !== 'default' &&
           ownerState.iconColor !== ownerState.color && {
-            color: undefined,
+            color: 'inherit',
```

Comment:
@siriwatknp This change doesn't work, now it doesn't overwrite the default color from line 132 anymore.

Reply:
My bad. It is fixed, see https://codesandbox.io/s/elastic-jasper-lhupwb?file=/index.tsx.

Reply:
Seems to work now 👍